setup.cfg

```diff
@@ -19,7 +19,11 @@ classifiers =
 
 [options]
 packages = find:
-install_requires = Click
+install_requires =
+    Click
+    pyyaml
+    neurodocker
+    testkraken
```

Comment:
@satra - but I've added `testkraken` here. should this be removed?

Reply:
this is fine. since it will be required for `nfm test`. my issues were with `nfm build`